src/App.tsx

```diff
@@ -6,10 +6,16 @@ import Welcome from './Welcome/Welcome';
 import Home from './Home/Home';
 import Sidebar from './Sidebar/Sidebar';
 import AppContainer from './AppContainer';
+import { CurrentUser } from './Teams/Teams';
 
-class App extends React.Component<any, any> {
-  constructor(props: Object) {
-    super({});
+interface AppProps {
```

Comment:
I think you can just call these `Props`.Like if you end up exporting them then you can always specify the exact type of props in whatever file you import them into.